service/db/dgclient.go

```diff
@@ -260,8 +273,51 @@ func (s DGClient) GetAllByClusterAndType(meta string, cluster string) (map[strin
 	return m, nil
 }
 
-//GetSchema - get all predicates
-func (s DGClient) GetSchema() ([]*api.SchemaNode, error) {
+//GetCacheContainsDBSchema - Get cache which contains db schema
+func (s DGClient) GetCacheContainsDBSchema() (*lru.Cache, error) {
+	//Add db schema to the cache
+	if InitLruCacheDBSchema == false {
+		dbSchemaNodes, err := s.GetSchemaFromDB()
+		if err != nil {
+			log.Errorf("err: %v", err)
+			return nil, err
+		}
+		LruCache.Add(CacheKey, dbSchemaNodes)
```

Comment:
@Lucy911, why does it make sense to always use the same `CacheKey` value? is it the case that you are trying to keep the 5 most recent schema values around and you're not going to ever store anything else in the cache?

Reply:
For now there is no other things to store in the cache.